api/processor_api.py

```diff
@@ -274,6 +274,50 @@ def handle_config_response(config, success):
                 )
             return JSONResponse(content=humps.decamelize(config))
 
+        def get_areas():
+            config = extract_config()
+            areas_name = [x for x in config.keys() if x.startswith("Area")]
+            return [map_area(x, config) for x in areas_name]
+
+        def reestructure_areas(config_dict):
+            """Ensure that all [Area_0, Area_1, ...] are consecutive"""
+            areas_names = [x for x in config_dict.keys() if x.startswith("Area")]
```

Comment:
I think that we should sort ascending the the area_names before the `for`.
For example, if for some reason the config file has the areas sorted descending, this metohd will change the index for all the areas.